PhpAmqpLib/Message/AMQPMessage.php

```diff
@@ -56,5 +53,7 @@ public function __construct($body = '', $properties = null)
     public function setBody($body)
     {
         $this->body = $body;
+
+        return $this;
```

Comment:
The phpdocs said it returned $this.  So I just made it true.  Personally I like fluent API for setters.  But that's just my opinion.